### DISCUSSION TITLE
2024-08-06 회의록

### DISCUSSION THREAD

**Discussion by eunohhh** [nbc-final-a10/TripBuddy#206] [회의록]
### 🔥 MVP 후 2주간 이어서 해결해야 할 내용들

- 남은 기능들(알림 / 팔로우 / 찜하기 / 스토리 /  온도지수 / 리치텍스트에디터 검색)
- 리팩토링
- 탠스택쿼리로 전환(인밸리데이트쿼리)
- 디렉토리(컴포넌트) 구조 정리
- 데스크탑 버전

### 🔥 유저 테스트관련

- 구글폼 작성(완)
- 계획 수립
- 슬랙메시지 작성(유상)

### 🔥 UI 관련

- 회사 정보 알려주는 푸터 없음 (우리가 텍스트 정리해서 전달)
- 폰트 관련 (프리텐다드 - 설정 오은)
- 컬러 다쓰는지 여부 (스타일 가이드 참고 - 민지님께서 설정)
- 없는 페이지 (전체보기(랭킹)페이지들, 알림 페이지 - 내용전달요망)
- 튜토리얼 (데탑에서도 비슷하게 가는거로) 

### 🔥 개발 관련

- UT 를 대비해서 : 연타 방지
- UT 를 대비해서 : 유효성 검사
- 8/7 저녁에 UT 예정

### 🔥 UT (8/7 저녁회의까지 준비해야 할 사항)
- 글쓰기 유효성검사(내일까지)
- 스토리 작성 페이지 완료하기(내일까지)
- 내일 저녁에 머지하고 테스트하고 슬랙 공지
- 182,400 원에 맞추고, 슬랙 공지문 준비

### 🔥 MVP 후 2주간 이어서 해결해야 할 내용들

- 남은 기능들(찜하기, 좋아요, 팔로우, 알림)
  - 스토리 / 찜하기 / 좋아요 - 오은 
  - 리치텍스트에디터 / 팔로우 - 김병준
  - 알림(팔로우찜하기되는대로) - 김민지
  - 검색 - 곽현정
- 리팩토링 / 컨벤션
  - 탠스택쿼리로 전환(인밸리데이트쿼리)
- 디렉토리(컴포넌트) 구조 정리
- 데스크탑 버전

### 🔥 차후 일정 점검(projects)
- 8/7 저녁 UT 회의 후 8/8(모레) 아침 회의에서 집중적으로 논의

### 🔥 티켓 일정 재분배
- 검색 어떻게 할지(단순검색 / 옵션검색  =>> 피드백 사항 : 피드백은 다 선택해야 된다고 생각하신 것 같은데, 실제로는 옵션이니 UI 에서 이런점을 보여주기)
  - 일단 >> 현행유지
- Image with src "/images/tuto0.webp" has legacy prop "layout". Did you forget to run the codemod?(튜토리얼)

### 🔥 실패했을 경우의 화면?
- **무언가 오류 error.tsx / not-found.tsx**
- 병준님은 우선 얼러트로

## 🍀 공통 트러블 슈팅

- 내용

// 이미지 테이블
<table>
  <tr>
    <th>이미지 1번 제목</th>
    <th>이미지 2번 제목</th>
  </tr>
  <tr>
    <td><img src="이미지 주소" width="300" height="300"/></td>
    <td><img src="이미지 주소" width="300" height="300"/></td>
  </tr>
</table>

**Comment by Captain-Kim** [comment]
## 김병준

- 와이어프레임에 없는 내용 추가 : 현재 와이어프레임 상에는 글쓰기 성공 화면만 있음.
